src/server/services/service/creating/creating.test.js

```diff
@@ -20,7 +20,8 @@ jest.mock(
   '~/src/server/deploy-service/helpers/fetch/fetch-available-versions.js'
 )
 
-describe('Services', () => {
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip('Services', () => {
```

Comment:
is it worth adding a TODO in here as to why it skipped?

Reply:
can do, thought the commit message was enough. hopefully you'll roll the entire commit back this week ;-)